x-pack/plugins/security_solution/public/entity_analytics/components/asset_criticality/use_asset_criticality.ts

```diff
@@ -14,7 +14,16 @@ import type { AssetCriticalityRecord } from '../../../../common/api/entity_analy
 import type { EntityAnalyticsPrivileges } from '../../../../common/api/entity_analytics/common';
 import type { AssetCriticality } from '../../api/api';
 import { useEntityAnalyticsRoutes } from '../../api/api';
-import { buildCriticalityQueryKeys } from './common';
+
+// SUGGESTION: @tiansivive Move this to some more general place within Entity Analytics
```

Comment:
moved from common.ts as it is only used here (and was the last thing left in that file)